scripts/run-depcheck.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# This script runs depcheck on two different packages: 'jaeger-ui' and 'plexus'
+
+# Run depcheck for the 'jaeger-ui' package
```

Comment:
I think this is more complicated than it needs to be:
  * depcheck takes a `--config` arg so I assume the rc file does not need to be in the local directory or specifically named
  * if so, you can use `tempfile()` to create two config files and not worry about deleting them (will happen automatically)
  * then you can use `set -ex` so that if depcheck returns a non-zero exit code the script will abort

with these changes I think your whole script can be just a few lines

Reply:
Thanks for this solution. Things are much simpler now 😄